skywalking-sniffer/skywalking-toolkit-activation/skywalking-toolkit-opentracing-activation/src/main/java/com/a/eye/skywalking/toolkit/activation/opentracing/span/interceptor/SpanSetTagInterceptor.java

```diff
@@ -30,6 +38,32 @@ else if (value instanceof Integer)
             ContextManager.activeSpan().setTag(key, value.toString());
     }
 
+    /**
+     * Fetch tag key of {@link Span#setTag}.
+     *
+     * @return tag key
+     */
+    private String fetchTagKeyFromArguments(Object[] arguments) {
```

Comment:
This method do not just fetch the tag key, you are changing it by specific circumstances. So wrong method name.